shared/src/main/java/cn/edu/tsinghua/iginx/utils/JsonUtils.java

```diff
@@ -0,0 +1,21 @@
+package cn.edu.tsinghua.iginx.utils;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+
+import java.nio.charset.StandardCharsets;
+
```

Comment:
Why use a Json parser different from that used in other modules?
Consider changes.